lib/MiniDroneBtAdapter.js

```diff
@@ -294,9 +376,22 @@ class MiniDroneBtAdapter extends EventEmitter {
             // I don't know why this needs some time
             setTimeout(() => this.emit('connected'), 200);
         });
+
     }
 
     /**
+     * Updates Rssi to get signal strenght
+     */
+     updateRssi() {
+       if (!this.peripheral) {
+           return;
+       }
+       this.peripheral.updateRssi((error, rssi) => {
+         this.emit('rssiUpdate', rssi);
```

Comment:
What happens when noble returns an error here? Should we emit `false` or not emit at all?